### PR TITLE
[zh-cn]: create the translation of webRTC `RTCPeerConnectionIceEvent` introduction

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnectionIceEvent

### Related issues and pull requests

* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/rtcpeerconnectioniceevent/index.md
* Last commit: https://github.com/mdn/content/commit/acfe8c9f1f4145f77653a2bc64a9744b001358dc
https://github.com/mdn/translated-content/issues/16473

### CODE REVIEW

File: files/zh-cn/web/api/rtcpeerconnectioniceevent/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: RTCPeerConnectionIceEvent
+slug: Web/API/RTCPeerConnectionIceEvent
+l1on:
```
Comment (T34-active) [inline files/zh-cn/web/api/rtcpeerconnectioniceevent/index.md:4]:
```suggestion
l10n:
```

File: files/zh-cn/web/api/rtcpeerconnectioniceevent/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [WebRTC](/zh-CN/docs/Web/API/WebRTC_API)
+- 它通常的目标是：{{domxref("RTCPeerConnection")}}。
```
Comment (T34-active) [inline files/zh-cn/web/api/rtcpeerconnectioniceevent/index.md:53]:
```suggestion
- 通常的目标是：{{domxref("RTCPeerConnection")}}。
```

File: files/zh-cn/web/api/rtcpeerconnectioniceevent/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+_{{domxref("RTCPeerConnectionIceEvent")}} 是 {{domxref("Event")}}，该事件还实现了这些属性。_
+
+- {{domxref("RTCPeerConnectionIceEvent.candidate")}} {{ReadOnlyInline}}
+  - : 包含与事件相关联的 {{domxref("RTCIceCandidate")}} 候选项，如果此事件指示没有更多的候选项，则为 `null`。
+
+## 构造函数
+
+- {{domxref("RTCPeerConnectionIceEvent.RTCPeerConnectionIceEvent()", "RTCPeerConnectionIceEvent()")}}
+  - : 返回一个新的 `RTCPeerConnectionIceEvent`。它接收两个参数，第一个是表示事件类型的字符串；第二个包含它所引用的 {{domxref("RTCIceCandidate")}} 的字典。
```
Comment (T34-active) [inline files/zh-cn/web/api/rtcpeerconnectioniceevent/index.md:26]:
```suggestion
  - : 返回新的 `RTCPeerConnectionIceEvent`。它接收两个参数，第一个是表示事件类型的字符串；第二个包含它所引用的 {{domxref("RTCIceCandidate")}} 的字典。
```

File: files/zh-cn/web/api/rtcpeerconnectioniceevent/index.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+## 实例方法
+
+_{{domxref("RTCPeerConnectionIceEvent")}} 是 {{domxref("Event")}}，因此该事件也实现了以下属性。没有特定的 {{domxref("RTCDataChannelEvent")}} 方法。_
+
+## 示例
+
+```js
+pc.onicecandidate = (ev) => {
+  console.log(
+    `交互式连接候选地址 (传输地址：“${ev.candidate.candidate}”）已添加到连接中。`,
```
Comment (yin1999) [inline files/zh-cn/web/api/rtcpeerconnectioniceevent/index.md:37]:
```suggestion
    `交互式连接候选地址（传输地址：“${ev.candidate.candidate}”）已添加到连接中。`,
```

**Review by yin1999 — APPROVED** [review]
感谢 @T34-active

**Pull request merged**